### DISCUSSION TITLE
MessengerBundle permissions?

### DISCUSSION THREAD

**Discussion by NiklasBr** [coreshop/CoreShop#2607] [Q&A] [answered]
Which permissions do we need to enable to allow non-admin users to view the menu item and dashboard?

**Comment by dpfaffenbauer** [comment]
Currently it is only available for admin. We can introduce a permission to further define that if needed

**Reply by NiklasBr** [reply]
Would love that. We are sending a lot of data to be imported into the system into the message queue. Product managers responsible for those imports would like to see an overview of the progress.

**Reply by dpfaffenbauer** [reply, marked as answer]
I created a issue and we'll implement that

**Reply by lukemacausland** [reply]
Hi @dpfaffenbauer, we were actually looking for this as well, and noticed you merged an update for this. When do you expect the next release to be, and do you plan to include this with it?

**Reply by dpfaffenbauer** [reply]
probably around beginning of may. you can use the dev-version for now: `composer require coreshop/messenger-bundle:4.0.x-dev`
